README.rst

```diff
@@ -0,0 +1,40 @@
+FogLAMP "random" C++ South plugin
+
+Build
+----
+To build FogLAMP Random C++ South plugin:
+- mkdir build
+- cd build
+- cmake ..
+
+- By default the FogLAMP develop package header files an libraries
+  are in use: /usr/include/foglamp and /usr/lib/foglamp
+- If FOGLAMP_ROOT env var is set and no -D options are set,
+  the header files and libraries paths are set to FOGLAMP_ROOT dir
```

Comment:
"are set to FOGLAMP_ROOT dir" -> "are pulled from the ones under the FOGLAMP_ROOT directory. Please note that you must first do a make in the FOGLAMP_ROOT directory."